lib/widgets.php

```diff
@@ -137,6 +137,7 @@ function gutenberg_get_legacy_widget_settings() {
 		foreach ( $wp_widget_factory->widgets as $class => $widget_obj ) {
 			$available_legacy_widgets[ $class ] = array(
```

Comment:
Right now I'm seeing "Gutenberg block" as an available legacy widget I guess we need to filter it ?